corehq/apps/reports/standard/monitoring.py

```diff
@@ -334,15 +336,46 @@ def make_column(title, help_text, num_days):
     def selected_users(self):
         return _get_selected_users(self.domain, self.request)
 
+    @property
+    def has_group_filters(self):
```

Comment:
nit: should we use "case_group" instead of group unless specifically referring to groups?
It does seem to get confusing at times if we use only "group" for case group and groups in HQ.

Reply:
I can see how this might get mistaken for the `Group` Django model. I like your suggestion of `case_group`. I'll change this out.

Reply:
[640f2f4](https://github.com/dimagi/commcare-hq/pull/34464/commits/640f2f49b479157073eff52dd49e1c5f92dc0c87)